### PR TITLE
Fix typo and grammar on README.md

### PR DESCRIPTION
**Pull request by yankeexe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ and transposed convolutions in the decoder.
 
 [Conv-Tasnet](https://arxiv.org/abs/1809.07454)
 is a separation model developed for speech which predicts a mask on a learnt over-complete linear representation
-using a purely convolutional model with stride of 1 and dilated convolutional blocks.
+using a purely convolutional model with the stride of 1 and dilated convolutional blocks.
```
Comment (adefossez) [inline README.md]: should be `a`

File: README.md [hunk 2]
```diff
@@ -11,16 +11,16 @@ and transposed convolutions in the decoder.
...
 repository and added support for multiple audio channels.
 
 
-When trained only on MusDB, Conv-Tasnet achieves higher SDR than Demucs (5.7 vs 5.6).
+When trained only on MusDB, Conv-Tasnet achieves higher SDR than Demucs (5.7 vs. 5.6).
 However, the audio it generates has significant artifacts as measured by human evaluations
 (MOS is 3.2 for Demucs, 2.9 for Conv-Tasnet). When trained with extra training data,
 Demucs and Conv-Tasnet obtain the same SDR. See [our paper][demucs_arxiv] Section 6 for more details or listen to our
-[audio samples][audio] .
+[audio samples][audio].
```
Comment (adefossez) [inline README.md]: This will make the `.` disappear because it is just next to a link. The space is not displayed by Markdown.

File: README.md [hunk 3]
```diff
@@ -59,22 +59,22 @@ for more details.
...
     conda activate demucs
 
-This will create a `demucs` environmnent with all the dependencies installed.
+This will create a `demucs` environment with all the dependencies installed.
 
 ### Using Windows
 
 If you are using Windows, replace `python3` by `python.exe` in all the commands provided hereafter :)
-Parts of the code are untested on Windows (in particular, training a new model). Please open an issue in case you have a problem.
+Parts of the code is untested on Windows (in particular, training a new model). Please open an issue in case you have a problem.
```
Comment (adefossez) [inline README.md]: Parts is plural, this should stay to `are` or `parts` should be changed to `part`.

File: README.md [hunk 4]
```diff
@@ -104,26 +104,26 @@ The models will be stored in the `models` folder. The list of pre-trained models
...
 ## Examining the results from the paper experiments
 
 The metrics for our experiments are stored in the `results` folder. In particular
-`museval` json evaluations are stored in `results/evals/EXPERIMENT NAME/results`.
+`museval` JSON evaluations are stored in `results/evals/EXPERIMENT NAME/results`.
 You can aggregate and display the results using
 ```bash
 python3 valid_table.py -p # show valid loss, aggregated with multiple random seeds
-python3 result_table.py -p # show SDR on test set, aggregated with multiple random seeds
+python3 result_table.py -p # show SDR on a test set, aggregated with multiple random seeds
```
Comment (adefossez) [inline README.md]: Should be `the test set`

File: README.md (hunk 1 above)
Comment (yankeexe) [reply README.md]: yes right.

File: README.md (hunk 2 above)
Comment (yankeexe) [reply README.md]: No it renders fine in markdown.

**Comment by yankeexe** [comment]
@adefossez I have a new commit, please check it out.

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Music Source Separation in the Waveform Domain
 
 We provide an implementation of Demucs and Conv-Tasnet for music source separation on the [MusDB][musdb] dataset.
-They can separate drums, bass and vocals from the rest with state-of-the-art results, surpassing previous waveform or spectrogram based methods.
+They can separate drums, bass, and vocals from the rest with state-of-the-art results, surpassing previous waveform or spectrogram based methods.
```
Comment (parcerend) [inline README.md]: I think it was ok without the comma here

File: README.md (hunk 3 above)
Comment (parcerend) [inline README.md]: "Parts" isn't a word for singular

File: README.md (hunk 5 above)
Comment (yankeexe) [reply README.md]: I think it is a general rule of thumb to add comma before `and` when we are addressing multiple items.

File: README.md (hunk 3 above)
Comment (yankeexe) [reply README.md]: Here the "Parts" represent 'mass noun' I guess, so `is` would be the right word. But I am not sure, what do you suggest @parcerend ?

File: README.md (hunk 5 above)
Comment (parcerend) [reply README.md]: It's ok, too

File: README.md (hunk 3 above)
Comment (parcerend) [reply README.md]: I think "are" was ok here
but tbh, my english is non-native

**Comment by adefossez** [comment]
Will close this PR because it is outdated.

**Pull request closed without merging**